### PR TITLE
allow the user to view the .dat file name in object selection menu

### PR DESCRIPTION
**Pull request by LeftofZen**
#1688

### CODE REVIEW

**Comment by LeftofZen** [comment]
fixes #1688

**Comment by LeftofZen** [comment]
looks like this
image

**Comment by AaronVanGeffen** [comment]
> looks like this image

The last two lines do not seem to correspond to any code in this PR. Did you remove them?

**Comment by LeftofZen** [comment]
@AaronVanGeffen oh yep I removed them as duncan requested, discussed in #coding on discord

**Review by duncanspumpkin — APPROVED** [review]

File: data/language/en-GB.yml [hunk 1]
```diff
@@ -2317,3 +2317,4 @@ strings:
   2262: "{NEWLINE}Length:  {INT32_1DP} tile(s)"
   2263: "{COLOUR WINDOW_2}{UINT16}/{UINT16}"
   2264: "Giant screenshot"
+  2265: "{COLOUR WINDOW_2} File Name: {COLOUR BLACK}{STRINGID}"
```
Comment (AaronVanGeffen) [inline data/language/en-GB.yml:2320]:
```suggestion
  2265: "{COLOUR WINDOW_2} Filename: {COLOUR BLACK}{STRINGID}"
```

**Review by AaronVanGeffen — APPROVED** [review]

**Pull request merged**